.github/workflows/updatecli.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: updatecli
```

Comment:
I like that the name matches the filename. However, I wonder if we can find something that describes what the workflow actually does ("create pr in xy repo", "bump x in y repo"). On the other hand, a comment might be enough.